src/test/java/org/circle8/integration/recorrido/RecorridoPostInicioTest.java

```diff
@@ -0,0 +1,33 @@
+package org.circle8.integration.recorrido;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import io.restassured.RestAssured;
+
+import org.circle8.ApiTestExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(ApiTestExtension.class)
+public class RecorridoPostInicioTest {
+
+	@Test
+	void testPostOk() {
+		RestAssured.given()
+					  .post("/recorrido/1/inicio")
+					  .then()
+					  .statusCode(200);
+		;
```

Comment:
estaria bueno chequear que en la DB realmente se haya cambiado la fecha de inicio

Reply:
Pero no veo la forma de verificarlo. Si genero una fecha para contrastar siempre va a tener un margen de error con lo que efectivamente se grabe, y ese margen de error puede hacer que se caiga el test.

Reply:
en la DB haces un `SELECT` para chequear que el `Recorrido` con `ID = 1` tenga una `FechaInicio IS NOT NULL`

Reply:
Para simplificar pregunté en el body si incluye el campo.